src/main/java/org/fcrepo/importexport/importer/Importer.java

```diff
@@ -387,7 +390,27 @@ private FcrepoResponse importContainer(final URI uri, final Model model) throws
             logger.debug("Using Bagit checksum ({}) for file ({})", checksum, containerFile.getPath());
             builder = builder.digest(checksum);
         }
-        return builder.preferLenient().perform();
+        final FcrepoResponse response = builder.preferLenient().perform();
+        if (response.getStatusCode() == 410 && config.overwriteTombstones()) {
+            deleteTombstone(response);
+            return builder.preferLenient().perform();
```

Comment:
I'm glad I wrote that test, because it didn't work.  So I wound up updating the tests to test the content of both containers and binaries, and found that I needed to create new PutBuilders (not just reset the body).  I'll push a new commit with that update.